Library/OptimizelyExperiment.swift

```diff
@@ -27,6 +27,6 @@ extension OptimizelyExperiment {
       .variant(
         for: OptimizelyExperiment.Key.nativeProjectPageCampaignDetails,
         userAttributes: optimizelyUserAttributes(with: project, refTag: refTag)
-      )
```

Comment:
I think the function return value can also be changed to `OptimizelyExperiment.Variant` if we're coalescing?

Reply:
Ah thanks - I had it like that but then reverted to right a failing test and forgot to switch back 😅